crates/flightrepl/src/lib.rs

```diff
@@ -60,10 +68,24 @@ pub async fn run(repl_config: ReplConfig) -> Result<(), Box<dyn std::error::Erro
         }
         match line {
             ".exit" | "exit" | "quit" | "q" => break,
+            ".error" => {
```

Comment:
We should distinguish between user query errors and runtime errors. I think this means query error, so we should label it as such.

Reply:
DuckDB labels their error as `Error: Parser Error`... I'd say the first error is redundant though. So, "Parser Error:" or "Query Error:"

<img width="582" alt="image" src="https://github.com/spiceai/spiceai/assets/80174/be980f36-a344-47fc-9d7b-5f584784f757">

Reply:
This is for both runtime errors and user query errors. I was intending this to be used as a debugging tool, so we could get access to the raw error without pasting it to the screen by default. I can hide the help text for it if we don't want to expose it.

Reply:
Renamed to `Query Error`